runtime/jcl/common/shared.c

```diff
@@ -71,7 +71,7 @@ typedef struct URLElements {
 } URLElements;
 
 
-static J9ClassPathEntry* getCachedURL(JNIEnv* env, jint helperID, const char* pathChars, jsize pathLen, UDATA cpeType, U_16 cpeStatus);
+static J9ClassPathEntry* getCachedURL(JNIEnv *env, jint helperID, const char *pathChars, jsize pathLen, UDATA cpeType, U_16 cpeStatus, char *correctedPath);
```

Comment:
Space before, not after `*`, please.
The final parameter should be `const char *correctedPath`; that will necessitate improving the signature of `createURLEntry()`:
```c
static jint createURLEntry(JNIEnv *env, jint helperID, J9ClassPathEntry **cpEntry_, const char *correctedPathCopy, UDATA cpeType, U_16 cpeStatus);
```